src/components/navigation-header/components/home-page-content/index.tsx

```diff
@@ -48,12 +49,17 @@ const HomePageHeaderContent = () => {
 
 	return (
 		<div className={sharedNavStyles.leftSide}>
-			<div className={sharedNavStyles.contentBeforeNav}>
-				<InlineSvg
-					className={s.siteLogo}
-					src={require('../../img/logo-white.svg?include')}
-				/>
-			</div>
+			<Link href="/">
+				<a
+					aria-label="HashiCorp Developer Home"
+					className={sharedNavStyles.contentBeforeNav}
+				>
+					<InlineSvg
+						className={s.siteLogo}
+						src={require('../../img/logo-white.svg?include')}
+					/>
+				</a>
+			</Link>
```

Comment:
Can we keep this wrapped in a div in case we need to add more stuff in it in the future?

```suggestion
			<div className={sharedNavStyles.contentBeforeNav}>
				<Link href="/">
					<a aria-label="HashiCorp Developer Home">
						<InlineSvg
							className={s.siteLogo}
							src={require('../../img/logo-white.svg?include')}
						/>
					</a>
				</Link>
			</div>
```

Reply:
Personally I don't have a strong opinion here, and would be fine with it either way — if we need to add stuff in the future, we can always add the wrapper `<div />` then 👍

Reply:
I'm not super opinionated so I'll add it back!